client/network_create_opts.go

```diff
@@ -0,0 +1,19 @@
+package client
+
+import "github.com/moby/moby/api/types/network"
+
+// NetworkCreateOptions holds options to create a network.
+type NetworkCreateOptions struct {
+	Driver     string                   // Driver is the driver-name used to create the network (e.g. `bridge`, `overlay`)
+	Scope      string                   // Scope describes the level at which the network exists (e.g. `swarm` for cluster-wide or `local` for machine level).
+	EnableIPv4 *bool                    `json:",omitempty"` // EnableIPv4 represents whether to enable IPv4.
+	EnableIPv6 *bool                    `json:",omitempty"` // EnableIPv6 represents whether to enable IPv6.
```

Comment:
For the client options, we can remove the `json` tags, as these will never be marshalled to JSON. MAYBE we can even get rid of the pointer values, but we can look at those as part of the functional arguments work.